sbin/build.sh

```diff
@@ -82,7 +82,9 @@ configuringBootJDKConfigureParameter()
 # Configure the boot JDK
 configuringMacOSCodesignParameter()
 {
-  addConfigureArgIfValueIsNotEmpty "--with-macosx-codesign-identity=" "\"${BUILD_CONFIG[MACOSX_CODESIGN_IDENTITY]}\""
+  if [ ! -z "${BUILD_CONFIG[MACOSX_CODESIGN_IDENTITY]}" ]; then
+    addConfigureArg "--with-macosx-codesign-identity=" "\"${BUILD_CONFIG[MACOSX_CODESIGN_IDENTITY]}\""
```

Comment:
It seems strange that this function uses `\"` and thus, cannot use `addConfigureArgIfValueIsNotEmpty`. Perhaps worth explaining in a comment as to why that is?